### PR TITLE
updated example

### PR DESCRIPTION
**Pull request by pinpong**

### CODE REVIEW

File: flutter_vlc_player/example/pubspec.yaml [hunk 1]
```diff
@@ -1,19 +1,40 @@
-name: flutter_vlc_player_example
-description: Demonstrates how to use the flutter_vlc_player plugin.
-publish_to: 'none'
+name: fluttervlcplayerexample
+description: A new Flutter project.
```
Comment (illia-romanenko) [inline flutter_vlc_player/example/pubspec.yaml:2]: It doesn't look right - should be reverted?

**Review by illia-romanenko — COMMENTED** [review]
Hi @pinpong! Thanks for the PR and updated example. Do you mind looking at it a bit more to reduce changelist and see whether all changes make sense?

Also I noticed that you might be using master vs stable that we used before - is there any particular reason for that - or can we just change example using stable channel?

Thanks in advance!

File: flutter_vlc_player/example/pubspec.yaml (hunk 1 above)
Comment (pinpong) [reply flutter_vlc_player/example/pubspec.yaml:2]: You are right. Let me check that.

Comment (pinpong) [reply flutter_vlc_player/example/pubspec.yaml]: Reason for the update was the error: `The lower bound of "sdk: '>=2.3.0 <3.0.0'" must be 2.12.0 or higher to enable null safety.
`
Creating a new example by `flutter create --org software.solid flutter_vlc_player_example` was the best option.
The new example (optional but recommend) needs also  few updates e.g. the build tools or gradle version.

Comment (mitchross) [reply flutter_vlc_player/example/pubspec.yaml]: Hmm. You dont need to regenerate a project just to update the SDK bounds. This project already supported null safety per https://github.com/solid-software/flutter_vlc_player/pull/205/files 


Im all for updated examples, but killing and regenerating the example project is completely unnecessary and this change should be reverted.  You can make all the gradle / xcode changes without regen.

Comment (pinpong) [reply flutter_vlc_player/example/pubspec.yaml]: The link you mention just updated the plugin and not the example.

Comment (mitchross) [reply flutter_vlc_player/example/pubspec.yaml]: Ah I am following now... Ok this makes sense.

**Comment by solid-oleksandrhordiienko** [comment]
Thanks for PR @pinpong!
I think that better to return functionality buttons (subtitles, language...) for page with a single video. Example must show all possible interactions, in my opinion. Added screenhots.
(New example)
image
(Old example)
image
(Functionality buttons, that I mentioned)
image

**Comment by mitchross** [comment]
> Thanks for PR @pinpong! I think that better to return functionality buttons (subtitles, language...) for page with a single video. Example must show all possible interactions, in my opinion. Added screenhots. (New example) image (Old example) image (Functionality buttons, that I mentioned) image

I agree alot of people complain in issues on "how to do something" having all the functionality in one screen helps folks out.

**Comment by pinpong** [comment]
Already fixed but it was a bug that already exists

**Comment by illia-romanenko** [comment]
Hey @pinpong, we merged new lints & related fixes, do you mind looking at all PRs and fix conflicts?

**Pull request closed without merging**